spec.bs

```diff
@@ -1686,16 +1683,26 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. Set |result|["{{URLPatternInit/hash}}"] to |hash|.
   1. Let |baseURL| be null.
   1. If |init|["{{URLPatternInit/baseURL}}"] is not null:
+    <div class="note">
+      The base URL can be used to supply additional context, but for each component, if |init| includes a component which is at least as specific as one in the base URL, none is inherited.
+
+      A component is more specific if it appears later in one of the following two lists (which are very similar to the order they appear in the URL syntax):
```

Comment:
Is this notion of "more specific" shared by both inheritance schemes? If so, maybe this helpful note should be moved somewhere more generic? Then the discussions of "later" can link to it.

However, they do seem subtlely different, so probably there's not much to be done here. (I guess, is it possible to give a similar nice list-based explanation for the constructor string parsing?)

Reply:
LGTM